search/search-client-query-transformer/src/main/java/io/camunda/search/clients/transformers/filter/AuthorizationFilterTransformer.java

```diff
@@ -22,13 +28,14 @@ public final class AuthorizationFilterTransformer
   @Override
   public SearchQuery toSearchQuery(final AuthorizationFilter filter) {
     return and(
-        longTerms("ownerKey", filter.ownerKeys()),
-        filter.ownerType() == null ? null : term("ownerType", filter.ownerType()),
-        stringTerms("permissions.resourceIds", filter.resourceKeys()),
-        filter.resourceType() == null ? null : term("resourceType", filter.resourceType()),
+        longTerms(OWNER_KEY, filter.ownerKeys()),
+        filter.ownerType() == null ? null : term(OWNER_TYPE, filter.ownerType()),
+        stringTerms("%s.%s".formatted(PERMISSIONS, PERMISSIONS_RESOURCEIDS), filter.resourceKeys()),
```

Comment:
❓  A field named "resourceIds" is mapped to "resourceKeys". In general convention in the API a key is a Long value and an ID is a String value.

Shouldn't the resourceKey be a resourceId?

Reply:
indeed, good catch!
It seems that it was updated everywhere-else except in the filter, I will update it

Reply:
Fixed @stoerti